pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -238,15 +238,8 @@ func (ev *Evaluator) findCandidates(ctx context.Context, allNodes []*framework.N
 		return nil, nil, err
 	}
 
-	offset, numCandidates := ev.GetOffsetAndNumCandidates(int32(len(potentialNodes)))
-	if loggerV := logger.V(5); logger.Enabled() {
-		var sample []string
-		for i := offset; i < offset+10 && i < int32(len(potentialNodes)); i++ {
-			sample = append(sample, potentialNodes[i].Node().Name)
-		}
-		loggerV.Info("Selected candidates from a pool of nodes", "potentialNodesCount", len(potentialNodes), "offset", offset, "sampleLength", len(sample), "sample", sample, "candidates", numCandidates)
```

Comment:
maybe we still want to log the number of potential nodes, numCandidates, and offset?

Reply:
Good input, the offset values a little I think because it's a random value, but maybe the numCandidates gives necessary information when profiling the preemption, so I added them all together in dryRun function together with the length of the PDBs.